access/templates/access/_error_summary.html

```diff
@@ -5,14 +5,17 @@
 	<div class="error-summary__text">
 		<h4>{% trans "Submission failed" %}</h4>
 		<p>{% trans "Check your answers before trying again:" %}</p>
+		<ul>
 		{% for field in result.form %}
```

Comment:
The `for` and the following `if` is not indented. Alternatively,  the `<li>` is indented too much.

Reply:
You are correct, this was fixed